Finalized translations

kairstenfay opened the pull request:
Translate the SCAN results report into Spanish, Vietnamese, and Chinese (simplified).

kairstenfay committed:
Translate Spanish SCAN report

Add the Spanish translation for the SCAN results report. Make minor
formatting tweaks to align the header (identifier) info properly.

kairstenfay committed:
Duplicate SCAN English report for translations

Duplicate the SCAN results English report as a starting point for the
new Spanish, Vietnamese, and Chinese (simplified) translations.

kairstenfay committed:
Translate Vietnamese SCAN report

Change the order of package imports to import polyglossia last (per the
documentation's recommendations). Make a minor edit to the header
(identifier info) layout to make everything align evenly.

kairstenfay committed:
Translate Chinese (simplified) SCAN report

Install Chinese fonts. Tweak the header layout to align everything more
closely.

kairstenfay commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Update RoR Portal & RoR Results w/ PHSKC-approved translations](https://trello.com/c/iaS57pKI/529-update-ror-portal-ror-results-w-phskc-approved-translations)

joverlee521 reviewed (approved):
Looked great when I ran this locally 👍

tsibley commented on scan/report-es.tex:
```diff
@@ -0,0 +1,273 @@
+\begin{itemize}
+\item
+
+  \link{https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDcasepositive.pdf}{Patients
+  with confirmed or suspected COVID-19 (Pacientes con sospecha o confirmación de
+  la COVID-19)}
+
+\item
+
+  \link{https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDexposed.pdf}{Patients
+  who were exposed to a confirmed COVID-19 case (Pacientes que estuvieron
+  expuestos a un caso confirmado de la COVID-19)}
+
+\item
+
+  \link{https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDconcerned.pdf}{Unexposed
+  patients with COVID-19 symptoms (Pacientes que no estuvieron expuestos y que
+  tienen síntomas de la COVID-19)}
```
WA DOH provides these documents in many other languages than English, so I think these links should point to the appropriate language, when available. For example, the Spanish version of the first link is https://www.doh.wa.gov/Portals/1/Documents/1600/coronavirus/COVIDcasepositiveSPANISH.pdf.

I don't know if that URL pattern is consistent, but I believe the matrix of documents × languages is listed on this page: https://www.doh.wa.gov/Emergencies/NovelCoronavirusOutbreak2020/HealthEducation

kairstenfay replied:
I updated the translated documents for Spanish and Vietnamese. No translations were available for Chinese.

@tsibley Oddly enough, the latest commit that changes the URL of these links breaks the build. Do you have any idea why?

tsibley replied:
> No translations were available for Chinese.

Wow, wasn't expecting that.

> Oddly enough, the latest commit that changes the URL of these links breaks the build. Do you have any idea why?

In LaTeX, underscores are special chars, so the ones in the new URLs need to be escaped like `\_`. :roll_eyes:

kairstenfay replied:
Tricky LaTeX! It's now fixed. Thanks for the help.

tsibley commented on scan/report-es.tex:
```diff
@@ -0,0 +1,273 @@
+\lhead{\includegraphics[height=1in,clip]{logo}}
+\rhead{\pbox[b][1in][c]{\textwidth}{
+    \small
+    \textbf{Test Performed By}\\
```
This phrase isn't translated in any of the versions, but it seems like it should be. An oversight when copying, or are we missing that phrase?

kairstenfay replied:
This phrase was missing from the translations. It is present in all of the translation documents as-is (see the [relevant Trello card](https://trello.com/c/iaS57pKI)).

kairstenfay replied:
"Test performed by" is now translated in all three documents.

kairstenfay committed:
Update external links with available translations

Update the links to King County COVID-19 information to the appropriate
translations where possibe. Spanish and Vietnamese have translated info,
but note that Chinese does not.

kairstenfay committed:
Translate 'Test performed by'

Translate the phrase "Test performed by" which was previously ommitted.

kairstenfay commented:
Next steps:
- [x] Rebuild the image and push it up to Docker Hub (see README)
- [x] Update generate-pdfs in our backoffice automation to bump the image version it uses
